### PR TITLE
[Do not merge] Test JupyterLab-LSP on RTC

### PR DESCRIPTION
**Pull request by hbcarlos**
As discussed in #590 this PR is for testing the extension on [JupyterLab RTC](https://github.com/jupyter-rtc/jupyterlab)

Modification on GitHub actions:

- Disabled linting (was linting jupyterlab's source code after cloning)
- Remove tests with nodejs < 12

### CODE REVIEW

File: .github/workflows/job.test.yml [hunk 1]
```diff
@@ -54,17 +54,24 @@ jobs:
...
           miniforge-variant: Mambaforge
           use-mamba: true
 
+      - name: Checkout JupyterLab RTC
+        uses: actions/checkout@master
+        with:
+          repository: jupyter-rtc/jupyterlab
+          path: ./jupyterlab
+      - name: Install JupyterLab RTC
+        run: pip install ./jupyterlab/
```
Comment (jtpio) [inline .github/workflows/job.test.yml:73]: Is this enough do install from source, or do we also need a `jupyter lab build`?

Comment (krassowski) [reply .github/workflows/job.test.yml:73]: Good question. The screenshots captured during the build suggest that we are running on the development version (with Isabela's CSS update), see below. Also, `python -m pip` might fix the smoke on MacOS (not that it is important).

image

Comment (krassowski) [reply .github/workflows/job.test.yml:73]: We probably could script a test that opens Help → About and takes a screenshot.

Comment (jtpio) [reply .github/workflows/job.test.yml:73]: We could also check the new `docprovider` and `shared-packages` are correctly included.

**Comment by hbcarlos** [comment]
I changed the installation command to fix the smoke test and added the `--dev-mode` parameter to the `jupyter-lab` command (not sure if that's the right way to add a parameter, though)

File: .github/workflows/job.test.yml [hunk 2]
```diff
@@ -135,27 +134,33 @@ jobs:
...
-        run: mamba install -n test -c conda-forge setuptools wheel 'jupyterlab${{ matrix.lab }}' 'nodejs${{ matrix.nodejs }}'
+        run: mamba install -n test -c conda-forge setuptools wheel 'nodejs${{ matrix.nodejs }}'
+      
+      - name: Checkout JupyterLab RTC
+        uses: actions/checkout@master
+        with:
+          repository: jupyter-rtc/jupyterlab
+          path: ./jupyterlab
+      - name: Install JupyterLab RTC
+        run: python -m pip install ./jupyterlab/
```
Comment (jtpio) [inline .github/workflows/job.test.yml:155]: Maybe we could now install the latest `3.1.0a7` prerelease?

Comment (hbcarlos) [reply .github/workflows/job.test.yml:155]: Yep, I'll open a new PR for the prerelease and close this one.

**Comment by krassowski** [comment]
@hbcarlos could you rebase now please? We fixed an issue with tests on master.

**Comment by hbcarlos** [comment]
> @hbcarlos could you rebase now please? We fixed an issue with tests on master.

Sure, I'm on it.

**Comment by krassowski** [comment]
As RTC is now merged and working well with LSP I will close this one. Thank you for testing this out.

**Pull request closed without merging**